Makefile

```diff
@@ -145,12 +164,26 @@ hack-test-unit: hack-build-dev-tools
 hack-codegen: hack-build-dev-tools
 	$(DOCKER_CMD) make codegen
 
+# Build an image. Example usages:
+#
+# Build image (kargo:latest)
+#   make hack-build
+#
+# Push a multi-arch image to a personal repository (myusername/kargo:latest)
+#   make hack-build IMAGE_REPO=myusername/kargo IMAGE_PUSH=true
+#
+# Build image (local architecture) with the :dev image tag
+# 	make hack-build IMAGE_TAG=dev
+#
+# Build a linux/amd64 image with a docker build option to not re-use docker build cache
+# 	make hack-build IMAGE_PLATFORMS=linux/amd64 DOCKER_BUILD_OPTS=--no-cache
 .PHONY: hack-build
 hack-build:
-	docker build \
+	docker buildx build \
+		$(DOCKER_BUILD_OPTS) \
 		--build-arg GIT_COMMIT=$(shell git rev-parse HEAD) \
 		--build-arg GIT_TREE_STATE=$(shell if [ -z "`git status --porcelain`" ]; then echo "clean" ; else echo "dirty"; fi) \
-		--tag kargo:dev \
+		--tag $(IMAGE_REPO):$(IMAGE_TAG) \
```

Comment:
Beware: `make hack-build` will now produce `kargo:latest` instead of `kargo:dev`

Reply:
Nevermind. went back to dev for now